### PR TITLE
V3 Docs Workflow

### PR DESCRIPTION
**Pull request by noahlitvin**
See https://github.com/Synthetixio/js-monorepo/pull/265

### CODE REVIEW

File: packages/core-contracts/contracts/interfaces/IERC721Enumerable.sol [hunk 1]
```diff
@@ -3,10 +3,21 @@ pragma solidity ^0.8.0;
 
 import "./IERC721.sol";
 
-interface IERC721Enumerable {
+interface IERC721Enumerable is IERC721 {
+    /**
+     * @dev Returns the total amount of tokens stored by the contract.
+     */
     function totalSupply() external view returns (uint256);
```
Comment (dbeal-eth) [inline packages/core-contracts/contracts/interfaces/IERC721Enumerable.sol:10]: returns the number of individual tokens in existance for this contract

File: packages/core-modules/contracts/modules/NftModule.sol [hunk 2]
```diff
@@ -11,7 +11,7 @@ import "../storage/NftStorage.sol";
 
 import "../interfaces/INftModule.sol";
 
-contract NftModule is INftModule, ERC721, NftStorage, InitializableMixin, OwnableMixin {
+contract NftModule is INftModule, ERC721Enumerable, NftStorage, InitializableMixin, OwnableMixin {
```
Comment (dbeal-eth) [inline packages/core-modules/contracts/modules/NftModule.sol:14]: ooh this is beyond a doc change, its a code change. is this intentional? I can agree it the change in general.

File: packages/synthetix-main/contracts/interfaces/IAccountTokenModule.sol [hunk 3]
```diff
@@ -0,0 +1,144 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@synthetixio/core-modules/contracts/interfaces/INftModule.sol";
+
+/// @title Module for managing account token (NFT) and accounts, each account is represented by an NFT
+interface IAccountTokenModule is INftModule {
+    /// @notice mints a new token (NFT) with the "requestedAccountId" id owned by "owner". It can ol=nly be called by the system
```
Comment (dbeal-eth) [inline packages/synthetix-main/contracts/interfaces/IAccountTokenModule.sol:8]: typo

File: packages/synthetix-main/contracts/interfaces/IAccountTokenModule.sol [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+     * @dev Returns the owner of the `tokenId` token.
+     *
+     * Requirements:
+     *
+     * - `tokenId` must exist.
+     */
+    function ownerOf(uint256 tokenId) external view returns (address owner);
+
+    /**
+     * @dev Safely transfers `tokenId` token from `from` to `to`.
```
Comment (dbeal-eth) [inline packages/synthetix-main/contracts/interfaces/IAccountTokenModule.sol:26]: transfers `tokenId` token from `from` to `to` with additional checks.

**Review by dbeal-eth — CHANGES_REQUESTED** [review]
would be greati f you could add a CI step to generate the docs as well and export an artifact or site or something

**Review by ajsantander — APPROVED** [review]
LGTM

**Pull request merged**